### PR TITLE
File Tidy Up

### PR DESCRIPTION
**Pull request by MrRowey**
**Aibrain.lua**  -  Moves Unused imports to bottom of file
**cybranunits.lua** - Moves Unused imports to bottom of file & fix order of calculation
**defaultexpolions.lua** - Moves Unused imports to bottom of file, fix order of calculation & added local where needed.
**defaultunits.lua** - Moves Unused imports to bottom of file & added local where needed.
**EffectUtilties.lua** - Added Local where needed
**platoon.lua** - Moves Unused imports to bottom of file & Added Local where needed
**ScenarioFramework.lua** - Moves Unused imports to bottom of file & Added Local where needed
**ScenarioPlatoonAI.lua** - Moves Unused imports to bottom of file & Added Local where needed
**DumpQAVChecklist.lua** - Fixed Escape Characters
**TriggerManager.lua** - Fixed typo

### CODE REVIEW

File: lua/ScenarioFramework.lua [hunk 1]
```diff
@@ -1721,25 +1720,25 @@ function OperationCameraThread(location, heading, faction, track, unit, unlock,
             WaitSeconds(2.5)
         end
         if faction == 1 then -- UEF
-            marker = {
+            local marker = {
```
Comment (Garanas) [inline lua/ScenarioFramework.lua:1723]: This introduces an error, marker should be defined before the first if statement (`local marker`) and then populated in the if statements

Comment (Garanas) [reply lua/ScenarioFramework.lua:1723]: See also line 1747, where marker is now undefined.

File: lua/ScenarioPlatoonAI.lua [hunk 2]
```diff
@@ -1063,9 +1062,9 @@ function StartBaseTransports(platoon, data, aiBrain)
 
         -- Unload transports
         if type(data.LandingLocation) == 'string' then
-            cmd = platoon:UnloadAllAtLocation(ScenarioUtils.MarkerToPosition(data.LandingLocation))
+            local cmd = platoon:UnloadAllAtLocation(ScenarioUtils.MarkerToPosition(data.LandingLocation))
```
[the remark this answers is not in the document]
Comment (Garanas) [reply lua/ScenarioPlatoonAI.lua:1065]: as it gets used at line 1070

File: lua/defaultexplosions.lua [hunk 3]
```diff
@@ -135,7 +130,7 @@ end
 -- @param unit The unit to get the diameter of.
 function GetAverageBoundingXYZRadius(unit)
     local bp = unit:GetBlueprint()
-    return ((bp.SizeX or 0 + bp.SizeY or 0 + bp.SizeZ or 0) * 0.333)
+    return ((0 + bp.SizeY or bp.SizeX or 0 + bp.SizeZ or 0) * 0.333)
```
Comment (Garanas) [inline lua/defaultexplosions.lua:133]: Same as on line 115

File: lua/defaultexplosions.lua [hunk 4]
```diff
@@ -117,7 +112,7 @@ end
 -- @param unit The unit to get the diameter of.
 function GetAverageBoundingXZRadius(unit)
     local bp = unit:GetBlueprint()
-    return ((bp.SizeX or 0 + bp.SizeZ or 0) * 0.5)
+    return ((0 + bp.SizeZ or bp.SizeX or 0) * 0.5)
```
[the remark this answers is not in the document]
Comment (MrRowey) [reply lua/defaultexplosions.lua:115]: Okay :) this was what the VSCode Extension said it should be will revert

File: lua/platoon.lua [hunk 5]
```diff
@@ -4701,7 +4700,7 @@ Platoon = Class(moho.platoon_methods) {
             end
             if target and newtarget and not target.Dead and target:GetFractionComplete() == 1
             and SUtils.XZDistanceTwoVectorsSq(target:GetPosition(), basePosition) < T4Radius * T4Radius then
-                blip = target:GetBlip(armyIndex)
+                local blip = target:GetBlip(armyIndex)
```
Comment (Garanas) [inline lua/platoon.lua:4703]: Appears to be unused

**Pull request closed without merging**